access_requests.go

```diff
@@ -0,0 +1,240 @@
+package gitlab
+
+import (
+	"fmt"
+	"net/url"
+	"time"
+)
+
+// AccessRequest represents a access request for a group or project.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/access_requests.html
```

Comment:
Please refer to the `ce` docs, unless the API is only supported in `ee` (goes for all links on this page of course). Thanks!